web_boolean_button/README.rst

```diff
@@ -0,0 +1,98 @@
+==================
+Web Boolean Button
+==================
```

Comment:
Then for the last general comment: the naming of your PR. This is not a migration because this module was before not in OCA. So you can use: `[ADD][13.0] Web Boolean Button module` to make it easy for reviewers to see.

Then when in future you also decide to contribute the 12.0 version it would be `[MIG][12.0] Web Boolean Button from 13.0` (or BCP for backport)